pack/fhv/fhvc.json

```diff
@@ -165,10 +446,266 @@
+    "faction_code": "mythos",
+    "flavor": "…strangely, they didn't howl. They laughed.\n<cite>Dr. Rosa Marquez, Miskatonic Survey, 1926</cite>",
+    "health": 4,
+    "illustrator": "Helge C. Balzer",
+    "name": "Lupine Hybrid",
+    "pack_code": "fhvc",
+    "position": 145,
+    "quantity": 1,
+    "text": "<b>Spawn</b> – Outer Fields (east).\nRetaliate. Patrol (The Farmhouse).\n<b>Forced</b> – After Lupine Hybrid attacks: Heal 1 damage from it.",
+    "traits": "Creature. Monster. Mutated. Elite.",
+    "type_code": "enemy"
+  },
+  {
+    "code": "10645c",
+    "encounter_code": "the_longest_night",
+    "encounter_position": 24,
+    "enemy_damage": 1,
+    "enemy_evade": 1,
+    "enemy_fight": 3,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "flavor": "…strangely, they didn't howl. They laughed.\n<cite>Dr. Rosa Marquez, Miskatonic Survey, 1926</cite>",
+    "health": 4,
+    "illustrator": "Helge C. Balzer",
+    "name": "Lupine Hybrid",
+    "pack_code": "fhvc",
+    "position": 145,
+    "quantity": 1,
+    "text": "<b>Spawn</b> – Outer Fields (south).\nRetaliate. Patrol (The Farmhouse).\n<b>Forced</b> – After Lupine Hybrid attacks: Heal 1 damage from it.",
+    "traits": "Creature. Monster. Mutated. Elite.",
+    "type_code": "enemy"
+  },
+  {
+    "code": "10646",
+    "encounter_code": "the_longest_night",
+    "encounter_position": 25,
+    "enemy_damage": 1,
+    "enemy_evade": 3,
+    "enemy_fight": 3,
+    "enemy_horror": 2,
+    "faction_code": "mythos",
+    "flavor": "I suppose that, like any goat, it eats everything.\n<cite>Dr. Rosa Marquez, Miskatonic Survey, 1926</cite>",
+    "health": 5,
+    "illustrator": "Stephen Somers",
+    "name": "Capra Hybrid",
+    "pack_code": "fhvc",
+    "position": 146,
+    "quantity": 3,
+    "text": "<b>Spawn</b> – Outer Fields location with the fewest enemies.\nAlert. Patrol (The Farmhouse).\nCapra Hybrid ignores decoys.",
+    "traits": "Creature. Monster. Mutated. Elite.",
+    "type_code": "enemy"
+  },
+  {
+    "code": "10647a",
+    "encounter_code": "the_longest_night",
+    "encounter_position": 28,
+    "enemy_damage": 2,
+    "enemy_evade": 2,
+    "enemy_fight": 2,
+    "faction_code": "mythos",
+    "health": 3,
+    "illustrator": "Peter Polach",
+    "name": "Equine Hybrid",
+    "pack_code": "fhvc",
+    "position": 147,
+    "quantity": 1,
+    "text": "<b>Spawn</b> – Outer Fields (north).\nPatrol (The Farmhouse).\nEquine Hybrid ignores traps.\n<b>Forced</b> – When Equine Hybrid would take any amount of damage: Reduce that amount to 1.",
+    "traits": "Creature. Monster. Mutated.",
+    "type_code": "enemy"
+  },
+  {
+    "code": "10647b",
+    "encounter_code": "the_longest_night",
+    "encounter_position": 29,
+    "enemy_damage": 2,
+    "enemy_evade": 2,
+    "enemy_fight": 2,
+    "faction_code": "mythos",
+    "health": 3,
+    "illustrator": "Peter Polach",
+    "name": "Equine Hybrid",
+    "pack_code": "fhvc",
+    "position": 147,
```

Comment:
I wonder if for b and c we need to boost the position by 1 (so 148 and 149). I think one for @zzorba to test when he has this live (or you can, @fspoettel ?) as I think having multiple cards with the same position might make the cards display somewhat oddly? Not 100% sure on that though...

Reply:
Great, that's good to know.

Reply:
@drawntotheflame here is what it looks like:

![image](https://github.com/zzorba/arkhamdb-json-data/assets/1682504/11335665-1ea8-44ab-9096-88b5fcab834a)